curriculum/challenges/english/08-coding-interview-prep/rosetta-code/luhn-test-of-credit-card-numbers.md

```diff
@@ -0,0 +1,101 @@
+---
+id: 5ea28156e79528a9ab248f27
+title: Luhn test of credit card numbers
+challengeType: 5
+---
+
+## Description
+
+<section id='description'>
+The <a href="https://en.wikipedia.org/wiki/Luhn algorithm" about="_blank">Luhn test</a> is used by some credit card companies to distinguish valid credit card numbers from what could be a random selection of digits.
```

Comment:
The <a href="https://en.wikipedia.org/wiki/Luhn algorithm" target="_blank">Luhn test</a> is used by some credit card companies to distinguish valid credit card numbers from what could be a random selection of digits.